source/usage-examples/command.txt

```diff
@@ -0,0 +1,46 @@
+=============
+Run a Command
+=============
+
+.. default-domain:: mongodb
+
+You can run all raw database operations using the
+``MongoDatabase.runCommand()`` method. A raw database operation is a
+command you can execute directly on the MongoDB server CLI. Commands
```

Comment:
Nitpick: would prefer specificity on what commands you're referring to in "Commands include", e.g. "These commands include..."